plugins/search/src/components/SearchResult/SearchResult.tsx

```diff
@@ -126,7 +128,7 @@ export const SearchResult = ({ searchQuery }: SearchResultProps) => {
 
   const [filteredResults, setFilteredResults] = useState<SearchResults>([]);
 
-  const searchApi = new SearchApi(catalogApi);
+  const searchApi = new SearchApi(catalogApi, discoveryApi);
```

Comment:
Better make this a Utility API instead, https://backstage.io/docs/api/utility-apis#defining-custom-utility-apis